src/torrent_info.cpp

```diff
@@ -324,9 +324,9 @@ namespace {
 		bdecode_node const attr = dict.dict_find_string("attr");
 		if (attr)
 		{
-			for (int i = 0; i < attr.string_length(); ++i)
+			for (char const c : attr.string_value())
```

Comment:
this makes me a little bit nervous. This code is a bit sensitive because there are torrents out there with tens of thousands of files, and sessions with thousands of torrents that need loading on startup. I believe this will construct a `std::string`, no?
Isn't there a function to get a `string_view`?

Reply:
don't be :)
`bedecode::string_value()` returns a `string_view`